Doc/reference/datamodel.rst

```diff
@@ -1350,7 +1350,7 @@ Basic customization
 
    .. versionchanged:: 3.7
       ``object.__format__(x, '')`` is now equivalent to ``str(x)`` rather
-      than ``format(str(self), '')``.
+      than ``format(str(x), '')``.
```

Comment:
From looking over the [bpo issue](https://bugs.python.org/issue28974) and [PR](https://github.com/python/cpython/pull/506/), it looks like Serhiy got the above explanation and example implementation mixed up a bit. It could make sense if he were using an instance of `object`, such as `obj` with something like this:
```
``obj.__format__('') is now equivalent to ``str(self)`` rather
than ``format(str(self))``.
```

but when using the actual class `object` as the explanation is doing, it does not make as much sense IMO to refer to `self`.

So, I tentatively agree with the PR, but I would prefer if @serhiy-storchaka could give it a final look-over.